crypto/xform.c

```diff
@@ -1,6 +1,18 @@
 /****************************************************************************
  * crypto/xform.c
- * $OpenBSD: xform.c,v 1.61 2021/10/22 12:30:53 bluhm Exp $
```

Comment:
is it intentional to remove rcsid? it's not mentioned in the commit message.

Reply:
the information is preserved in the commit log

Reply:
it's ok if it's intentional.